src/Components/Results/Results.tsx

```diff
@@ -335,7 +335,7 @@ const Results = ({ handleTextFieldChange }: ResultsProps) => {
                   </p>
                   {navigator.assistance_link.default_message && (
                     <h4 className="font-weight">
-                      Link:{' '}
+                      <FormattedMessage defaultMessage="Link: " id="navigator.linkLabel" />
```

Comment:
What do you think about having the `id` listed first and then the `defaultMessage` on all of these?